src/webgl/light.js

```diff
@@ -494,6 +494,12 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
   return this;
 };
 
+p5.prototype.imageLight = function(img){
+  // setting activeImageLight so that the setUniform is executed
+  this._renderer.activeImageLight = img;
```

Comment:
oh I just noticed, I think right now this doesn't get reset in `push`/`pop`, so we'd need to add to the `push` method here to do something similar to this: https://github.com/processing/p5.js/blob/d713b35e79866c5bf79fbcf338ff8761da846248/src/webgl/p5.RendererGL.js#L1583-L1583
e.g.
```js
properties.activeImageLight = this.activeImageLight;
```

Reply:
done